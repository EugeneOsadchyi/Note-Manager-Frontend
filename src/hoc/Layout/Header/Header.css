.Header {
  border: 2px solid black;
  padding: 0 10px;
  position: fixed;
  top: 0;
  width: 100%;
  height: 34px;
  line-height: 30px;
}
