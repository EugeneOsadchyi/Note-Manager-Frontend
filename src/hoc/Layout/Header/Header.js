import React from 'react';
import PropTypes from 'prop-types';
import './Header.css';

const Header = ({ title }) => (
  <header className="Header">
    {title}
  </header>
);

Header.propTypes = {
  title: PropTypes.string.isRequired,
};

export default Header;
