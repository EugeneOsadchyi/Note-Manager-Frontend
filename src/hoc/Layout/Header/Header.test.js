import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Header from './Header';

describe('Header', () => {
  const title = 'Notes Manager';
  const header = shallow(<Header title={title} />);

  it('renders properly', () => {
    expect(shallowToJson(header)).toMatchSnapshot();
  });

  it('contains correct title', () => {
    expect(header.text()).toEqual(title);
  });
});
