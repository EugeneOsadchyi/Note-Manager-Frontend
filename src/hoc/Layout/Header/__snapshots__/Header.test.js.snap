// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders properly 1`] = `
<header
  className="Header"
>
  Notes Manager
</header>
`;
